Resync WPT quirks tests

rniwa opened the pull request:
#### 24698482dbececbd257e41fc8fbd27af84bc477e
<pre>
Resync WPT quirks tests
<a href="https://bugs.webkit.org/show_bug.cgi?id=244023">https://bugs.webkit.org/show_bug.cgi?id=244023</a>

Reviewed by Tim Nguyen.

Updated web platform tests for quirks mode as of 23ca5bfb4ac7a1b52355588cff4b0da47cffaae4.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/import-expectations.json:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/quirks/META.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/blocks-ignore-line-height-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/blocks-ignore-line-height.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/body-fills-html-quirk-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/body-fills-html-quirk-float-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/body-fills-html-quirk-float.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/body-fills-html-quirk-inline-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/body-fills-html-quirk-inline.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/body-fills-html-quirk-positioned-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/body-fills-html-quirk-positioned.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/body-fills-html-quirk-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/body-fills-html-quirk-ref2.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/body-fills-html-quirk-vertical-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/body-fills-html-quirk-vertical.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/body-fills-html-quirk.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/classname-query-after-sibling-adoption-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/classname-query-after-sibling-adoption.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/crashtests/list-item-whole-line-quirks-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/crashtests/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/dd-dl-firefox-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/dd-dl-firefox-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/dd-dl-firefox-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/hashless-hex-color/limited-quirks-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/hashless-hex-color/limited-quirks.html: Copied from LayoutTests/imported/w3c/web-platform-tests/quirks/unitless-length/no-quirks.html.
* LayoutTests/imported/w3c/web-platform-tests/quirks/hashless-hex-color/no-quirks-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/hashless-hex-color/no-quirks.html: Copied from LayoutTests/imported/w3c/web-platform-tests/quirks/unitless-length/no-quirks.html.
* LayoutTests/imported/w3c/web-platform-tests/quirks/hashless-hex-color/quirks-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/hashless-hex-color/quirks.html: Copied from LayoutTests/imported/w3c/web-platform-tests/quirks/unitless-length/no-quirks.html.
* LayoutTests/imported/w3c/web-platform-tests/quirks/hashless-hex-color/support/common.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/hashless-hex-color/support/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/hashless-hex-color/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/historical/list-item-bullet-size-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/historical/list-item-bullet-size-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/historical/list-item-bullet-size.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/historical/vertical-align-in-quirks-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/historical/vertical-align-in-quirks.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/historical/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/html-fills-viewport-quirk-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/html-fills-viewport-quirk-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/html-fills-viewport-quirk-vertical-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/html-fills-viewport-quirk-vertical.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/html-fills-viewport-quirk.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/line-height-calculation-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/line-height-calculation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/line-height-in-list-item.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/line-height-in-list-item.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/line-height-limited-quirks-mode-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/line-height-limited-quirks-mode.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/line-height-quirks-mode-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/line-height-quirks-mode.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/line-height-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/line-height-trailing-collapsable-whitespace-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/line-height-trailing-collapsable-whitespace-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/line-height-trailing-collapsable-whitespace.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/percentage-height-calculation-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/percentage-height-calculation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/percentage-height-quirk-excludes-flex-grid-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/percentage-height-quirk-excludes-flex-grid-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/percentage-height-quirk-excludes-flex-grid-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/percentage-height-quirk-excludes-flex-grid-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/reference/percentage-height-quirk-excludes-flex-grid-002-ref-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/reference/percentage-height-quirk-excludes-flex-grid-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/reference/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/supports-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/supports.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/table-cell-nowrap-minimum-width-calculation-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/table-cell-nowrap-minimum-width-calculation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/table-cell-width-calculation-abspos-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/table-cell-width-calculation-abspos.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/table-cell-width-calculation-applies-to-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/table-cell-width-calculation-applies-to.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/table-replaced-descendant-percentage-height-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/text-decoration-doesnt-propagate-into-tables/quirks-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/text-decoration-doesnt-propagate-into-tables/quirks-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/text-decoration-doesnt-propagate-into-tables/quirks.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/text-decoration-doesnt-propagate-into-tables/standards-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/text-decoration-doesnt-propagate-into-tables/standards-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/text-decoration-doesnt-propagate-into-tables/standards.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/text-decoration-doesnt-propagate-into-tables/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/unitless-length/no-quirks.html:
* LayoutTests/imported/w3c/web-platform-tests/quirks/w3c-import.log: Added.
* LayoutTests/tests-options.json:

Canonical link: <a href="https://commits.webkit.org/253548@main">https://commits.webkit.org/253548@main</a>
</pre>













<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/f085f7cf744b99f9ab9ebaabf31feed7c0d19a16

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/86315 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/69/builds/30236 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/17289 "Built successfully") | [❌ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/95164 "Hash f085f7cf for PR 3398 does not build") | [✅ 🛠 🧪 win](https://ews-build.webkit.org/#/builders/10/builds/148872 "Built successfully and passed tests") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/64/builds/28599 "Built successfully") | [✅ 🛠 mac-debug](https://ews-build.webkit.org/#/builders/71/builds/25258 "Built successfully") | [✅ ~~🛠 gtk~~](https://ews-build.webkit.org/#/builders/36/builds/78483 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/90420 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/91914 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/68/builds/23198 "Passed tests") | [✅ ~~🛠 mac-AS-debug~~](https://ews-build.webkit.org/#/builders/61/builds/73321 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/23300 "Passed tests") | 
| | [✅ ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/9/builds/78209 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/3/builds/78625 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [❌ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/66306 "Found 2 new API test failures: /WebKit2Gtk/TestWebKitAccessibility:/webkit/WebKitAccessibility/text/state-changed, /WebKit2Gtk/TestWebKitWebView:/webkit/WebKitWebView/is-web-process-responsive") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/66/builds/26558 "Built successfully") | [✅ ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/73/builds/12493 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/67/builds/26467 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/13505 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/63/builds/28143 "Built successfully") | [✅ ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/61/builds/73321 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/75/builds/973 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/65/builds/28082 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/32786 "Passed tests") | | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (approved):
r=me if ews is happy

rniwa committed:
Resync WPT quirks tests
https://bugs.webkit.org/show_bug.cgi?id=244023

Reviewed by Tim Nguyen.

Updated web platform tests for quirks mode as of 23ca5bfb4ac7a1b52355588cff4b0da47cffaae4.

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/import-expectations.json:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/quirks/META.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/blocks-ignore-line-height-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/blocks-ignore-line-height.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/body-fills-html-quirk-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/body-fills-html-quirk-float-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/body-fills-html-quirk-float.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/body-fills-html-quirk-inline-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/body-fills-html-quirk-inline.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/body-fills-html-quirk-positioned-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/body-fills-html-quirk-positioned.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/body-fills-html-quirk-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/body-fills-html-quirk-ref2.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/body-fills-html-quirk-vertical-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/body-fills-html-quirk-vertical.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/body-fills-html-quirk.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/classname-query-after-sibling-adoption-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/classname-query-after-sibling-adoption.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/crashtests/list-item-whole-line-quirks-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/crashtests/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/dd-dl-firefox-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/dd-dl-firefox-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/dd-dl-firefox-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/hashless-hex-color/limited-quirks-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/hashless-hex-color/limited-quirks.html: Copied from LayoutTests/imported/w3c/web-platform-tests/quirks/unitless-length/no-quirks.html.
* LayoutTests/imported/w3c/web-platform-tests/quirks/hashless-hex-color/no-quirks-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/hashless-hex-color/no-quirks.html: Copied from LayoutTests/imported/w3c/web-platform-tests/quirks/unitless-length/no-quirks.html.
* LayoutTests/imported/w3c/web-platform-tests/quirks/hashless-hex-color/quirks-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/hashless-hex-color/quirks.html: Copied from LayoutTests/imported/w3c/web-platform-tests/quirks/unitless-length/no-quirks.html.
* LayoutTests/imported/w3c/web-platform-tests/quirks/hashless-hex-color/support/common.js: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/hashless-hex-color/support/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/hashless-hex-color/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/historical/list-item-bullet-size-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/historical/list-item-bullet-size-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/historical/list-item-bullet-size.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/historical/vertical-align-in-quirks-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/historical/vertical-align-in-quirks.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/historical/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/html-fills-viewport-quirk-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/html-fills-viewport-quirk-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/html-fills-viewport-quirk-vertical-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/html-fills-viewport-quirk-vertical.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/html-fills-viewport-quirk.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/line-height-calculation-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/line-height-calculation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/line-height-in-list-item.tentative-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/line-height-in-list-item.tentative.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/line-height-limited-quirks-mode-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/line-height-limited-quirks-mode.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/line-height-quirks-mode-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/line-height-quirks-mode.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/line-height-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/line-height-trailing-collapsable-whitespace-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/line-height-trailing-collapsable-whitespace-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/line-height-trailing-collapsable-whitespace.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/percentage-height-calculation-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/percentage-height-calculation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/percentage-height-quirk-excludes-flex-grid-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/percentage-height-quirk-excludes-flex-grid-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/percentage-height-quirk-excludes-flex-grid-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/percentage-height-quirk-excludes-flex-grid-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/reference/percentage-height-quirk-excludes-flex-grid-002-ref-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/reference/percentage-height-quirk-excludes-flex-grid-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/reference/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/supports-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/supports.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/table-cell-nowrap-minimum-width-calculation-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/table-cell-nowrap-minimum-width-calculation.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/table-cell-width-calculation-abspos-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/table-cell-width-calculation-abspos.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/table-cell-width-calculation-applies-to-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/table-cell-width-calculation-applies-to.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/table-replaced-descendant-percentage-height-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/text-decoration-doesnt-propagate-into-tables/quirks-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/text-decoration-doesnt-propagate-into-tables/quirks-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/text-decoration-doesnt-propagate-into-tables/quirks.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/text-decoration-doesnt-propagate-into-tables/standards-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/text-decoration-doesnt-propagate-into-tables/standards-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/text-decoration-doesnt-propagate-into-tables/standards.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/text-decoration-doesnt-propagate-into-tables/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/quirks/unitless-length/no-quirks.html:
* LayoutTests/imported/w3c/web-platform-tests/quirks/w3c-import.log: Added.
* LayoutTests/tests-options.json:

Canonical link: https://commits.webkit.org/253548@main
